### DISCUSSION TITLE
Error: In order to prevent potential loops, that email cannot be used.

### DISCUSSION THREAD

**Discussion by domattioli** [anonaddy/anonaddy#397] [Q&A]
When I created my account, I setup my DuckDuckGo email (call it `Duck_1`) as the recipient, which strips my emails of trackers and then forwards it to my gmail (call it `Gmail_1`). I had no problems with this for months, until yesterday's tinkering. I mistakenly removed my `Duck_1` as a recipient but now when I try to re-add it I get the error:

```Error: In order to prevent potential loops, that email cannot be used.```

The only recipient I have right now is to a non-gmail, work email account (call it `Work_1`) that is entirely separate from `Duck_1` - there is no forwarding between the accounts. All of my AnonAddy aliases' recipient are set to `Work_1`. I created a second Duckmail account (`Duck_2`) and set it to forward to a separate, dummy gmail account (`Gmail_2`). In summary, my Frankenstein's Monster setup of emails for debugging this are:

- `Duck_1` forwarding to `Gmail_1`. The latter may be set as a recipient in AnonAddy, and the former _used to_ be able of being a recipient, but no longer.
- `Work_1` is currenly the sole registered recipient on my account. All AnonAddy aliases are set to forward to this account.
- `Duck_2` was created for debugging.
- `Work_2` was created for debugging.
- `Duck_2` was setup to forward to `Gmail_2`. The latter may be set as a recipient, but the former may not.

Any ideas on how to resolve this?

**Comment by willbrowningme** [comment]
Please [send me an email](https://anonaddy.com/contact/) and I'll look into this for you.

**Comment by willbrowningme** [comment]
Hi @domattioli please could you respond regarding the payment dispute that you've opened?

If you are no longer using the service I will happily provide a full refund.
